website/docs/migration/family_and_dispose/family_and_dispose.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:math';
+
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+import '../utils.dart';
+
+part 'family_and_dispose.g.dart';
+
+/* SNIPPET START */
+@riverpod
+class BugsEncounteredNotifier extends _$BugsEncounteredNotifier {
+  late String _id;
+  @override
+  FutureOr<int> build(String featureId) {
+    _id = featureId;
```

Comment:
Don't do this.

There's already a this.featureId generated

Reply:
Woops!! I didn't know about this. My bad.

Reply:
BUT I *have to* do this in the non-generated version.

Reply:
No the non-generated one has this.argument/arg, whatever the name is

Reply:
Wow!